post-types/templates/page-profiles-with-sub-navigation.php

```diff
@@ -0,0 +1,50 @@
+  <section class="container">
```

Comment:
@Huyeng In general, this PR looks good to me. HOWEVER, I do not see how you are integrating the submenu that editors can now specify on the profile's configuration.

I would expect some line of code where the variable **_page_with_sub_navigation**  is actually used. Did you forget to add it?